.travis.yml

```diff
@@ -4,8 +4,7 @@ language: groovy
 
 jdk:
   - oraclejdk8

```

Comment:
It does, many are still using Java 7. Remember that Oracle aren't the only ones distributing JDKs. OpenJDK is available from several vendors and go back as far as JDK 6.